test/cluster_test.go

```diff
@@ -498,6 +498,34 @@ func UpdateTLSClusterTest(ctx goctx.Context) {
 					aeroCluster.Spec.AerospikeConfig.Value["network"] = network
 					err = updateCluster(k8sClient, ctx, aeroCluster)
 					Expect(err).Should(HaveOccurred())
+
```

Comment:
Need to test:
1. Non-tls to tls update in single step -> Should deny
2. Tls to non-tls update in single step -> Should deny